spotmarketrequest_test.go

```diff
@@ -0,0 +1,141 @@
+package packngo
+
+import (
+	"log"
+	"testing"
+	"time"
+)
+
+func TestAccSpotMarketRequestBasic(t *testing.T) {
+	// This test is only going to create the spot market request with
+	// max bid price set to half of current spot price, so that the devices
+	// are not run at all.
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, projectID, teardown := setupWithProject(t)
+	defer teardown()
+
+	ps := InstanceParameters{
+		BillingCycle:    "hourly",
+		Plan:            "baremetal_0",
+		OperatingSystem: "rancher",
+		Hostname:        "test{{index}}",
```

Comment:
Perhaps make host name recognizable for tests like we do for devices.

Reply:
fixed